rpcs3/Emu/Cell/PPUFunction.cpp

```diff
@@ -570,7 +570,7 @@ extern std::string ppu_get_syscall_name(u64 code)
 	case 985: return "sys_dbg_get_console_type";
 	}
 
-	return fmt::format("syscall_%llu", code);
+	return fmt::format("%llu", code);
```

Comment:
These strings are used in LLVM, you shouldn't just edit them.

Reply:
hmm ok